### PR TITLE
improve-image-tag-recommendation

### PR DESCRIPTION
**Pull request by vitorvezani**
This PR fixes #

## Checklist
* [X] I have signed the CLA
* [X] I have updated/added any relevant documentation

### What's the goal of this PR?
Improve image tags recommendations

### CODE REVIEW

File: plugins/trivy/pkg/image/newest_images.go [hunk 1]
```diff
@@ -54,22 +60,47 @@ func fetchTags(ctx context.Context, imageRepoName string) ([]string, error) {
...
+	for _, tag := range suggestedTags {
+		v, err := semver.NewVersion(tag)
+		if err != nil {
+			logrus.Infof("could not parse tag %s: %v", tag, err)
+			continue
+		}
+
+		cPre := curVersion.Prerelease()
+		vPre := v.Prerelease()
+		isKnownPreRelease := lo.Contains(knownPreReleases, cPre)
```
Comment (rbren) [inline plugins/trivy/pkg/image/newest_images.go:85]: I don't think we need to check `knownPreReleases` - just that the two match

**Review by jdesouza — APPROVED** [review]

File: plugins/trivy/pkg/image/newest_images.go (hunk 1 above)
Comment (vitorvezani) [reply plugins/trivy/pkg/image/newest_images.go:85]: If we remove that, `v0.0.16-alpine` will not be recommended for `v0.0.14-beta1`

Comment (vitorvezani) [reply plugins/trivy/pkg/image/newest_images.go]: fixed, I guess that was intended

File: plugins/trivy/pkg/image/version.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	tags := []string{}
+	for _, v := range s {
+		tags = append(tags, v.Original())
+	}
+	return tags
+}
+
+// Sort sorts the given slice of Version
+func Sort(versions []*semver.Version) {
+	sort.Sort(Versions(versions))
```
Comment (rbren) [inline plugins/trivy/pkg/image/version.go:33]: Why the `Versions()` part of this?

**Review by rbren — APPROVED** [review]

File: plugins/trivy/pkg/image/version.go (hunk 2 above)
Comment (vitorvezani) [reply plugins/trivy/pkg/image/version.go:33]: `sort.Sort` requires a [sort.Interface](https://pkg.go.dev/sort#Interface) implementation, `Versions` is the struct that implements that

**Pull request merged**